### PR TITLE
Replace even older service location

### PR DESCRIPTION
**Pull request by nemchik**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [ ] I have read the [contributing](https://github.com/linuxserver/docker-swag/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

### CODE REVIEW

**Review by thespad — APPROVED** [review]
Maybe wait for the "old files" PR on the base to merge before merging this so it gets included.

**Comment by nemchik** [comment]
You mean this one? https://github.com/linuxserver/docker-baseimage-alpine-nginx/pull/126

**Comment by thespad** [comment]
Yes

**Pull request merged**
